src/event-listeners/onAnnotationSelected.js

```diff
@@ -18,4 +18,5 @@ export default dispatch => (e, annotationList, action) => {
     });
     dispatch(actions.expandNotes(ids));
   }
+  $(document).trigger('annotationSelected', {e, annotationList, action});
```

Comment:
I think that this is maybe a little overzealous and triggers too much but I want to limit functional changes so I've left it as Glen implemented it.